core/block_svg.js

```diff
@@ -639,6 +639,20 @@ Blockly.BlockSvg.prototype.onMouseDown_ = function(e) {
     // Click, not drag, so stop waiting for other touches from this identifier.
     Blockly.Touch.clearTouchIdentifier();
   } else if (!this.isMovable()) {
+    // Click handler for immovable blocks.
+    var mouseupWrapper = Blockly.bindEvent_(document,
```

Comment:
This should be bindEventWithChecks_.  Have you checked that this works on touch, especially with multi-touch?